samples/places-autocomplete-addressform/src/index.njk

```diff
@@ -13,53 +13,46 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-{% extends '../../../shared/layout.njk'%} {% block html %}
-<div id="locationField">
-  <input
-    id="autocomplete"
-    placeholder="Enter your address"
-    onFocus="geolocate()"
-    type="text"
-  />
-</div>
-
-<!-- Note: The address components in this sample are typical. You might need to adjust them for
-               the locations relevant to your app. For more information, see
-         https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform
+{% extends '../../../shared/layout.njk'%}
+{% block external %}
+<link href="https://fonts.googleapis.com/css?family=Roboto:400,500" rel="stylesheet">
+{% endblock %}
+{% block html %}
+<!-- Note: The address components in this sample are based on North American address format. You might need to adjust them for the locations relevant to your app. For more information, see
+https://developers.google.com/maps/documentation/javascript/examples/places-autocomplete-addressform
     -->
 
-<table id="address">
-  <tr>
-    <td class="label">Street address</td>
-    <td class="slimField">
-      <input class="field" id="street_number" disabled="true" />
-    </td>
-    <td class="wideField" colspan="2">
-      <input class="field" id="route" disabled="true" />
-    </td>
-  </tr>
-  <tr>
-    <td class="label">City</td>
-    <td class="wideField" colspan="3">
-      <input class="field" id="locality" disabled="true" />
-    </td>
-  </tr>
-  <tr>
-    <td class="label">State</td>
-    <td class="slimField">
-      <input class="field" id="administrative_area_level_1" disabled="true" />
-    </td>
-    <td class="label">Zip code</td>
-    <td class="wideField">
-      <input class="field" id="postal_code" disabled="true" />
-    </td>
-  </tr>
-  <tr>
-    <td class="label">Country</td>
-    <td class="wideField" colspan="3">
-      <input class="field" id="country" disabled="true" />
-    </td>
-  </tr>
-</table>
+<form action="" method="get" autocomplete="off">
+  <span class="title">Sample address form for North America</span>
+  <label class="fullfield" for="gmp-a1">
+    <!-- Avoid the word "address" in id, name, or label text to avoid browser autofill from conflicting with Place Autocomplete. Star or comment bug https://crbug.com/587466 to request Chromium to honor autocomplete="off" attribute. -->
```

Comment:
as someone who uses browser autofill and loves it, I beg you to make this work with browser autofill.

I agree https://crbug.com/587466 is a pain, but encouraging address forms to block this is not in the best interests of users in the generic use of an address form, where a user wanting delivery to their house is a perfectly valid use case. (I believe that's why chromium killed off autocomplete=off - developers over-applied it.)

Reply:
done